<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AskSkelDialog</class>
 <widget class="QDialog" name="AskSkelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenBRF</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSkelFrom">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>cbSkelTo</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSkelTo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>From (current skeleton):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>To (destination skeleton):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Change geometry of meshes currently rigged for a skeleton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>26</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>to make it follow a new skeleton:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>151</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Put result in:</string>
   </property>
   <widget class="QRadioButton" name="radioButtonB2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>same mesh (overwrite)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonB1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>32</y>
      <width>84</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>new mesh</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonB0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>new vertex-ani frame</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Method:</string>
   </property>
   <widget class="QRadioButton" name="radioButtonA0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>generic</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonA1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>84</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>humanoids</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>cbSkelFrom</tabstop>
  <tabstop>cbSkelTo</tabstop>
  <tabstop>radioButtonA0</tabstop>
  <tabstop>radioButtonA1</tabstop>
  <tabstop>radioButtonB0</tabstop>
  <tabstop>radioButtonB1</tabstop>
  <tabstop>radioButtonB2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AskSkelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AskSkelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
